Dockerfile

```diff
@@ -71,6 +71,6 @@ ADD . /app
 RUN pip3 install -r /app/requirements.txt -t /app/lib/ --upgrade
 RUN pip3 install gunicorn==19.9.0
 
-ENV PYTHONPATH=/app:/app/apiv4:/app/lib:/app/IDC-Common:${PYTHONPATH}
+ENV PYTHONPATH=/app:/app/apiv1:/app/lib:/app/IDC-Common:${PYTHONPATH}
 
```

Comment:
This won't work; the folder is api not apiv1. So /app/api: and then below, api:app (unless you've changed it to apiv1 but it doesn't appear to be the case).